.rubocop_todo.yml

```diff
@@ -647,6 +647,8 @@ Style/PreferredHashMethods:
   Enabled: false
 
 # This cop supports safe auto-correction (--auto-correct).
+# Enabling this cop will remove raising RuntimeErrors and cause spec test
+# failures
```

Comment:
I think if the plan is to have this cop disabled indefinitely, we can probably move it from .rubocop_todo.yml to .rubocop.yml to indicate that it's not in some kind of pending state.